### PR TITLE
(xDai) Activate POSDAO

### PR DESCRIPTION
**Pull request by varasev**
The `spec.json` is changed for `xDai` chain to activate POSDAO consensus.

The changed spec requires a validator to have [Parity Ethereum v2.7.2-posdao-stable](https://github.com/poanetwork/open-ethereum/releases/tag/v2.7.2-posdao-stable) node.

The activation will occur at block `#9186425` (1-Apr-2020 ~08:18 am UTC).

### CODE REVIEW

**Review by vbaranov — COMMENTED** [review]
@varasev could you please verify new contracts in Blockscout? I just installed 3.1.1 to xDai Blockscout instance.

**Comment by varasev** [comment]
> @varasev could you please verify new contracts in Blockscout? I just installed 3.1.1 to xDai Blockscout instance.

@vbaranov I've just verified them 👍

File: contracts.json [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-  "METADATA_ADDRESS": "0xeb35547e1e4630fc6ed9a6f3d4c5e436422f63ea",
-  "PROXY_ADDRESS": "0x9a14adf50abd5a05e274650c5c6da3e93b48180b",
-  "POA_ADDRESS": "0x22e1229a2c5b95a60983b5577f745a603284f535",
-  "EMISSION_FUNDS_ADDRESS": "0xff9c96fdb2aebf16b294775a0723d3b7f1914162",
-  "REWARD_BY_BLOCK_ADDRESS": "0x867305d19606aadba405ce534e303d0e225f9556"
-}
+  "ERC677BridgeTokenRewardable": "0x38A541381D1167b6ACE528eb33adA252ebFBb15E",
+  "ValidatorSetAuRaCode": "0x7B48d8474c9d05a9AF4323c012Cb0E7F4C276673",
+  "ValidatorSetAuRaProxy": "0xB87BE9f7196F2AE084Ca1DE6af5264292976e013",
+  "StakingAuRaCode": "0x365F19752F3deBAF29DCC61D5001AA4f311d700e",
```
Comment (vbaranov) [inline contracts.json:5]: Just curious, is this property intentionally named `StakingAuRaCode`? I see, that the name of the contract is `StakingAuRa`. Btw, I see mismatches for multiple contracts.

**Review by igorbarinov — APPROVED** [review]

**Review by vbaranov — APPROVED** [review]
Approved, just one curiosity comment

File: contracts.json (hunk 1 above)
Comment (varasev) [reply contracts.json:5]: That's fine: the lines in this JSON don't define contract names. These are just designations to see addresses of upgradable contracts (`*Code` is an implementation contract address, `*Proxy` is a proxy contract).

**Pull request merged**